Added ruffle

monkeyx-net opened the pull request:
Added support for ruffle. Just removed the commented the default settings  for ruffle.
 
 On branch retrodeck-ruffle
 Changes to be committed:
	modified:   resources/systems/linux/es_find_rules.xml
	modified:   resources/systems/linux/es_systems.xml

monkeyx-net committed:
Added ruffle

 On branch retrodeck-ruffle
 Changes to be committed:
	modified:   resources/systems/linux/es_find_rules.xml
	modified:   resources/systems/linux/es_systems.xml

XargonWan commented on resources/systems/linux/es_find_rules.xml:
```diff
@@ -810,6 +810,7 @@
             <entry>ruffle</entry>
         </rule>
         <rule type="staticpath">
+            <entry>/app/bin/ruffle</entry>
```
I would put:

```xml
<entry>/app/bin/ruffle</entry> <!-- RetroDECK -->
```

To show it's our custom entry, it helps when we have to compare the incoming edits.

XargonWan commented:
A part of that comment stuff I believe it's good to go, just add that comment so we can merge :)

monkeyx-net committed:
Added the comment as suggested.

 On branch retrodeck-ruffle
 Changes to be committed:
	modified:   resources/systems/linux/es_find_rules.xml

XargonWan commented on resources/systems/linux/es_find_rules.xml:
```diff
@@ -810,10 +810,11 @@
             <entry>ruffle</entry>
         </rule>
         <rule type="staticpath">
+            <entry>/app/bin/ruffle</entry>
```
You said you're using a wrapper, is this the wrapper? Otherwise it will call the original Ruffle.
In case the wrapper should be recognizable, I usually use a naming convention: emulator-rd-wrapper, in this case ruffle-rd-wrapper

monkeyx-net replied:
I think I looked at what was done for Pico 8 and followed that example. Ie copy the wrapper to a new name in the manifest.

Following this suggestion is what I would have done if I had looked at Pico 8.

I'll change the es-de pr and yml in feat/ruffle.

I'll also pick a version of ruffle to use. Then I can remove it from the placeholder.

monkeyx-net committed:
The retrodeck comment was against the wrong entry pnm updating the static path.

 Now shown against the top entry and points to wrapper script

 On branch retrodeck-ruffle
 Changes to be committed:
	modified:   es_find_rules.xml
